test/functional.js

```diff
@@ -1362,7 +1360,7 @@ module.exports = {
                 const config = createWebpackConfig('web/build', 'dev');
                 config.addEntry('main', ['./css/roboto_font.css', './js/no_require', 'vue']);
                 config.addEntry('other', ['./css/roboto_font.css', 'vue']);
-                config.setPublicPath('http://localhost:8080/build');
+                config.setPublicPath('http://localhost:8080/custom_prefix');
```

Comment:
Hmm. This is a tough one. In my original version, I apparently designed it (I say "apparently" because this is what the test tells us, but it was something I was "debating" by myself for a bit) so that when the entrypoints.json file is dumped, if the user is using an absolute publicPath (with http://), that is NOT included in the entrypoints values. Instead, the user would use the value from `entrypoints.json` to make a second lookup in `manifest.json`.

I'm not sure that was the best idea. And so, the NEW implementation is better. Specifically: it looks like the new implementation always makes the value in `entrypoints.json` the final value - so there is no `manifest.json` lookup necessary. Do you guys see any problems with this? It would mean that nobody could use `entrypoint.json` to *simply* get the "local" paths of the needed CSS or JS files - it would always be the absolute path (with the `http://` if they have one set). I'm trying to think if that's a problem...

Anyways, if we assume that the new implementation is better, then there is just one change to make here: change the publicPath BACK to `http://localhost:8080/build` because it's useful to set this path to something different than the manifestKeyPrefix (it's just a better test). Then, update the test below to look for `http://localhost:8080/build` in the values. Also, the description of this test would need to be updated, or 2 tests would need to be created. The new description would be something like:

> Custom public path is used in entrypoints.json, but does not affect manifest.json

Reply:
@weaverryan I updated the test to what I think you meant? not quite sure. Let me know if it needs more changes.